ext/osx/puppet.plist

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+        <key>EnvironmentVariables</key>
+        <dict>
+                <key>PATH</key>
+                <string>/opt/puppetlabs/bin:/usr/bin:/bin:/usr/sbin:/sbin</string>
+                <key>RUBYLIB</key>
+		<string>/opt/puppetlabs/puppet/lib/ruby/site_ruby/2.1.0/</string>
```

Comment:
I'm not certain that RUBYLIB is required at all.  @haus?



> On May 4, 2015, at 23:15, Melissa <notifications@github.com> wrote:
> 
> In ext/osx/puppet.plist:
> 
> > @@ -0,0 +1,34 @@
> > +<?xml version="1.0" encoding="UTF-8"?>
> > +<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
> > +<plist version="1.0">
> > +<dict>
> > +        <key>EnvironmentVariables</key>
> > +        <dict>
> > +                <key>PATH</key>
> > +                <string>/opt/puppetlabs/bin:/usr/bin:/bin:/usr/sbin:/sbin</string>
> > +                <key>RUBYLIB</key>
> > +		<string>/opt/puppetlabs/puppet/lib/ruby/site_ruby/2.1.0/</string>
> hmm, is there anyway we could make this an erb and process it as we're installing it, so that we don't have to hardcode the ruby version here?
> 
> —
> Reply to this email directly or view it on GitHub.
>